python/paddle/fluid/tests/unittests/dygraph_to_static/test_basic_api_transformation.py

```diff
@@ -0,0 +1,440 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import numpy as np
+import paddle.fluid as fluid
+import unittest
+import inspect
+import gast
+
+from paddle.fluid.dygraph.jit import dygraph_to_static_output
+from paddle.fluid.dygraph.dygraph_to_static.utils import is_dygraph_api
+
+SEED = 2020
+np.random.seed(SEED)
+
+
+def dyfunc_to_variable(x):
+    input = fluid.dygraph.to_variable(x)
+    pool2d = fluid.dygraph.Pool2D(
+        pool_size=2, pool_type='avg', pool_stride=1, global_pooling=False)
+    res = pool2d(input)
+    return res
+
+
+class TestDygraphBasicAPI_to_variable(unittest.TestCase):
```

Comment:
Better to be:

TestDygraphBasicApi_ToVariable

Reply:
Thanks. Done.